### DISCUSSION TITLE
Facing Issue while Deploying on IIS

### DISCUSSION THREAD

**Discussion by mayankkshah** [serenity-is/Serenity#7260] [Help]
Can anyone help me with resovling the issue I am facing.
This is the error I am facing while deploying on IIS. 
![image (2)](https://github.com/user-attachments/assets/cc294521-518c-49fe-88a8-cfe7824cbee7)

Thankyou In advance!

**Comment by luongnv6** [comment]
You should check the event log, I think you haven't installed Net core hosting bundle
https://dotnet.microsoft.com/en-us/download/dotnet/8.0#:~:text=The%20ASP.NET%20Core%20Runtime%20enables%20you%20to%20run%20existing

**Reply by mayankkshah** [reply]
I have already Installed the Net core hosting bundle, and as per your suggestion I checked the event log I have attached the Image of my event log will you help me where am i getting wrong. 
![Event](https://github.com/user-attachments/assets/602e70e5-2acf-4039-8e19-d2b953d1786a)

Also now i am getting another issue: 
<img width="960" alt="error" src="https://github.com/user-attachments/assets/14838836-1070-402d-b13f-7f3c3a1fffb0">

Thankyou!

**Comment by luongnv6** [comment]
I think you just need to grant access to IIS AppPool as in the error, you can google this error
